scripts/build.js

```diff
@@ -10,6 +10,7 @@ EventEmitter.defaultMaxListeners = 100; // Increase the default limit to avoid m
 
 const myEmitter = new EventEmitter();
 
+let limitNumWorkers = false;
```

Comment:
i don't think you need this variable, just check if program.numWorkers > 0 to see if you need to apply a limit

Reply:
Fixed : https://github.com/ovh/manager/blob/4e79984e20ebc2f3cf687cdc8dfee89da6348d8a/scripts/build.js#L56